frictionless/detector.py

```diff
@@ -122,48 +124,181 @@ def __setattr__(self, name, value):
 
```

Comment:
Shall we remove `__setattr__` method as it's not used anymore?

Reply:
@roll I have removed it.